Cargo.toml

```diff
@@ -21,3 +21,10 @@ thiserror = "1.0.40"
 
 [features]
 serde = ["dep:serde", "ordered-float/serde", "indexmap/serde"]
+
+[dev-dependencies]
```

Comment:
Can this be changed to `[target.'cfg(serde)'.dev-dependencies]`?

Reply:
if you meant `'cfg(feature = "serde")'` according to [rust docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies) you'd have to use `[features]` section for that